ui_advanced_config.py

```diff
@@ -0,0 +1,220 @@
+        self.objectsLabel.grid(row=2,column=1,columnspan=2,sticky="nsew")
+        self.objectsIDLabel.grid(row=3,column=1,sticky="nsew")
+        self.objectsIDEntry.grid(row=3,column=2,sticky="nsew")
+        self.objectsNameLabel.grid(row=4,column=1,sticky="nsew")
+        self.objectsNameEntry.grid(row=4,column=2,sticky="nsew")
+        self.objectsFillAliveLabel.grid(row=5,column=1,sticky="nsew")
+        self.objectsFillAliveEntry.grid(row=5,column=2,sticky="nsew")
+        self.objectsFillDeadLabel.grid(row=6,column=1,sticky="nsew")
+        self.objectsFillDeadEntry.grid(row=6,column=2,sticky="nsew")
+        self.objectsTextLabel.grid(row=7,column=1,sticky="nsew")
+        self.objectsTextEntry.grid(row=7,column=2,sticky="nsew")
+        self.objectsHealthLabel.grid(row=8,column=1,sticky="nsew")
+        self.objectsHealthEntry.grid(row=8,column=2,sticky="nsew")
+        self.objectsDensityLabel.grid(row=9,column=1,sticky="nsew")
+        self.objectsDensityEntry.grid(row=9,column=2,sticky="nsew")
+        self.objectsCompIDsLabel.grid(row=10,column=1,sticky="nsew")
+        self.objectsCompIDsEntry.grid(row=10,column=2,sticky="nsew")
+        self.objectsPointsCountLabel.grid(row=11,column=1,sticky="nsew")
+        self.objectsPointsCountEntry.grid(row=11,column=2,sticky="nsew")
+        self.objectsUpdateButton.grid(row=10,column=1,columnspan=2)
+
+        self.mapsColumn.pack(side=tk.LEFT,fill=tk.BOTH,expand=True,padx=10,pady=10)
+        self.mapsLabel.grid(row=1,column=1,columnspan=2,sticky="nsew")
+        self.mapsIDLabel.grid(row=2,column=1,sticky="nsew")
+        self.mapsIDEntry.grid(row=2,column=2,sticky="nsew")
+        self.mapsNameLabel.grid(row=3,column=1,sticky="nsew")
+        self.mapsNameEntry.grid(row=3,column=2,sticky="nsew")
+        self.mapsEdgeObjIDLabel.grid(row=4,column=1,sticky="nsew")
+        self.mapsEdgeObjIDEntry.grid(row=4,column=2,sticky="nsew")
+        self.mapsDescLabel.grid(row=5,column=1,sticky="nsew")
+        self.mapsDescEntry.grid(row=5,column=2,sticky="nsew")
+        self.mapsWidthLabel.grid(row=6,column=1,sticky="nsew")
+        self.mapsWidthEntry.grid(row=6,column=2,sticky="nsew")
+        self.mapsHeightLabel.grid(row=7,column=1,sticky="nsew")
+        self.mapsHeightEntry.grid(row=7,column=2,sticky="nsew")
+        self.mapsUpdateButton.grid(row=8,column=1,columnspan=2)
+
+
+
+    def updateTeamsJSON(self): 
+        self.teamData[self.teamNameEntry.get()]['size'] = self.teamSizeEntry.get()
+        self.teamData[self.teamNameEntry.get()]['agent_defs'][0]['callsign'] = self.callsignEntry.get()
+        self.teamData[self.teamNameEntry.get()]['name'] = self.teamNameEntry.get()
+
+        print(self.teamNameEntry.get())
+        self.teamsJSON = json.dumps(self.teamData, indent=4)
+        print(self.teamsJSON)
+
+        with open('settings/teams.json', 'w') as f:
+            f.write(self.teamsJSON)
+        f.close()
+
+    
```

Comment:
Maybe add some TODO statements here to clarify this is an upcoming area of work in case somebody else wants to pick up the work